backend/src/api/model/event.rs

```diff
@@ -414,7 +420,7 @@ impl AuthorizedEvent {
                     select {event_cols}, \
                         row_number() over(order by ({sort_col}, id) {sort_order}) as row_num, \
                         count(*) over() as total_count \
-                    from events \
+                    from all_events as events \
```

Comment:
The two `with_omitted_table_prefix` calls also need to be adjusted in this function. I think? Apparently it works like this too, but I don't know why :D

Oh and: the API endpoint docs should say that deleted events are also returned.

Reply:
> The two `with_omitted_table_prefix` calls also need to be adjusted in this function.

The first call will not work with `all_events` without adjusting the `impl_from_db!` code to use `all_events` as well (which would break other things), and the second one doesn't seem to matter since `events` and `all_events` share all their columns.

Reply:
Oh yeah right... I confused something. Should be fine now.